plugins/okta/admin/src/lifecycles.js

```diff
@@ -0,0 +1,3 @@
+function lifecycles() {}
```

Comment:
Is this needed?

Reply:
Not needed currently but comes as a part of boilerplate.

Reply:
let's remove everything that's not needed to reduce clutter